src/test/java/org/fundacionjala/at15/katas/pokerhands/mauricio/PokerHandTest.java

```diff
@@ -0,0 +1,20 @@
+package org.fundacionjala.at15.katas.pokerhands.mauricio;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class PokerHandTest {
```

Comment:
Even if covered by tests from other classes, there should be unit tests for specific units of logic, that is, each method on PokerHand class. Try to add more of those here